ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -687,31 +696,68 @@ export default class ConfirmTransactionBase extends Component {
       () => {
         this._removeBeforeUnload();
 
+        ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
```

Comment:
I think adding code fences throughout the `handleSubmit` function makes this hard to read.

As an alternative, we could add another method to the component `handleMMISubmit`. Then, at the point of submission, we can decide whether to call `handleSubmit` or `handleMMISubmit` based on what build we are in. Perhaps this could be facilitated by renaming the current `handleSubmit` to `handleMainSubmit`, and then write a new `handleSubmit` which is just responsible for calling the appropriate `handle*Submit` method.

This might also allow us to eliminate code fencing from this file.

Reply:
Yes, you're right, I'll improve it with your suggestions. Thank you

Reply:
Without looking at it too exhaustively, my only concern with this approach is that MMI needs to execute the same code as MM "but" add extra code to it. So having two different methods could duplicate the code. Wdyt?

Reply:
I updated the code. Please, let me know if the current approach is the best.